clang/test/Interpreter/lambda.cpp

```diff
@@ -1,7 +1,10 @@
 // REQUIRES: host-supports-jit
 // UNSUPPORTED: system-aix
 // RUN: cat %s | clang-repl | FileCheck %s
-// RUN: cat %s | clang-repl -Xcc -O2 | FileCheck %s
+// At -O2, somehow "x = 42" appears first when piped into FileCheck,
+// see https://github.com/llvm/llvm-project/issues/143547.
+// RUN: %if !system-windows %{ cat %s | clang-repl -Xcc -Xclang -Xcc -verify -Xcc -O2 | FileCheck %s %}
```

Comment:
Can't we do: `XFAIL: system-windows`?

Reply:
`UNSUPPORTED` is fine.

Reply:
Sure, had to make my own branch to do that: https://github.com/llvm/llvm-project/pull/143893